docs/mysql.md

```diff
@@ -0,0 +1,60 @@
+# Using MySQL
+
+The Docker configuration in this repository defaults to using PostgreSQL.
+If you prefer to work with MySQL, some adjustments need to be made.
+
+First, install the doctrine/orm pack as described: `make composer c='req symfony/orm-pack`
+
+## Docker configuration
+Change the database image to use mysql instead of postgresql in `compose.yaml`:
+
+```yaml
+###> doctrine/doctrine-bundle ###
+  database:
+    image: 'mysql:8.0'
+    environment:
+      - MYSQL_ROOT_PASSWORD=r007_p4ssw0rd
+      - MYSQL_DATABASE=databasename
+      - MYSQL_USER=mysqluser
+      - MYSQL_PASSWORD=p4ssw0rd
+    volumes:
+      - database_data:/var/lib/mysql:rw
+###< doctrine/doctrine-bundle ###
+```
```

Comment:
```suggestion
```diff
###> doctrine/doctrine-bundle ###
-   image: postgres:${POSTGRES_VERSION:-15}-alpine
+   image: mysql:${MYSQL_VERSION:-8}
    environment:
-     POSTGRES_DB: ${POSTGRES_DB:-app}
+     MYSQL_DATABASE: ${MYSQL_DATABASE:-app}
      # You should definitely change the password in production
+     MYSQL_ROOT_PASSWORD: ${MYSQL_ROOT_PASSWORD:-!ChangeMe!}
-     POSTGRES_PASSWORD: ${POSTGRES_PASSWORD:-!ChangeMe!}
+     MYSQL_PASSWORD: ${MYSQL_PASSWORD:-!ChangeMe!}
-     POSTGRES_USER: ${POSTGRES_USER:-app}
+     MYSQL_USER: ${MYSQL_USER:-app}
    volumes:
-     - database_data:/var/lib/postgresql/data:rw
+     - database_data:/var/lib/mysql:rw
      # You may use a bind-mounted host directory instead, so that it is harder to accidentally remove the volume and lose all your data!
-     # - ./docker/db/data:/var/lib/postgresql/data:rw
+     # - ./docker/db/data:/var/lib/mysql:rw
###< doctrine/doctrine-bundle ###
```
```